cas-management-webapp/src/main/java/org/jasig/cas/services/web/beans/RegisteredServiceEditBean.java

```diff
@@ -182,30 +136,6 @@ public static RegisteredServiceEditBean fromRegisteredService(final RegisteredSe
 
         configureAccessStrategy(svc, bean);
 
-        configureOauthSettings(svc, bean);
-
-        bean.setTheme(svc.getTheme());
-        bean.setEvalOrder(svc.getEvaluationOrder());
-        configureLogoutTypeAndUrl(svc, bean);
-
-        final RegisteredServiceUsernameAttributeProvider provider = svc.getUsernameAttributeProvider();
-        final RegisteredServiceUsernameAttributeProviderEditBean uBean = bean.getUserAttrProvider();
-
-        configureUsernameAttributeProvider(provider, uBean);
-
-        configurePublicKey(svc, bean);
```

Comment:
`fromRegisteredService` which is now broken into smaller methods. I think it was more than 150 lines.